emData/download.sh

```diff
@@ -1,21 +1,30 @@
 #!/usr/bin/env bash
 
-#### fw_synch_210503 ####
-# Standard configuration
-memprints_url="https://cernbox.cern.ch/index.php/s/CipX7CfTXIj1lcK/download"
-luts_url="https://cernbox.cern.ch/index.php/s/UDSvClVZksBr1Pq/download"
-# Combined modules
-memprints_url_cm="https://www.dropbox.com/s/lf088lvyvg2t6jh/MemPrintsCombined_210319.tgz?dl=0"
-luts_url_cm="https://www.dropbox.com/s/legrvm3gyu5hrth/LUTsCombined_210319.tgz?dl=0"
-
-#### fw_synch_201005 ####
-#memprints_url="https://cernbox.cern.ch/index.php/s/y7IWeDG4x7Sg7Im/download"
+# fw_synch_201005
+#tarball_url="https://cernbox.cern.ch/index.php/s/y7IWeDG4x7Sg7Im/download"
 #luts_url="https://cernbox.cern.ch/index.php/s/DuhCjcykSHZLRhM/download"
 
-#### fw_synch_200515 ####
+# standard configurations - temporary
+tarball_url="https://www.dropbox.com/s/8942mkttj198y90/MemPrintsStandard_210315.tgz?dl=0"
```

Comment:
Is this an old version of the download script? I don't see the point of most changes in this file atm (like changing back to calling it tarball_url etc)

Reply:
Ok, I started back from the script in master, and made the minimal changes to get everything working.